### PR TITLE
[AudioManager] Add new APIs for device running changed event

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Added new APIs for device running state

### API Changes ###

- TCSACR-175
   (It's related native ACR-1147.)

Added:
- public bool IsRunning
- public static event EventHandler DeviceRunningChanged

Deprecated:
- public AudioDeviceState State
- public static event EventHandler DeviceStateChanged

### CODE REVIEW

File: src/Tizen.Multimedia/Interop/Interop.Device.cs [hunk 1]
```diff
@@ -53,6 +57,9 @@ internal static partial class AudioDevice
             [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_get_device_state_by_id")]
             internal static extern AudioManagerError GetDeviceState(int deviceId, out AudioDeviceState state);
 
+            [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_is_device_running")]
+            internal static extern AudioManagerError GetDeviceRunning(IntPtr device, out bool isRunning);
```
[the remark this answers is not in the document]
Comment (sangchul1011) [reply src/Tizen.Multimedia/Interop/Interop.Device.cs]: And I found that it need to declare deprecation to some enum values - Deactivated, Activated in AudioDeviceOptions.
Could you please update these also? (you can check these in AudioDeviceOptions.cs file)

[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia/Interop/Interop.Device.cs]: It's typo. I'll fix it.
I'll deprecate ```AudioDeviceState``` enum also.
But I didn't found native ACR related to AudioDeviceOptions deprecation. Please let me know.

[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia/Interop/Interop.Device.cs]: TCSACR-175 was accepted already. I'll create another JIRA for this deprecation.

[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia/Interop/Interop.Device.cs]: Fixed typo and uploaded.

**Pull request merged**